metaworld/envs/assets_v2/objects/assets/plug.xml

```diff
@@ -5,10 +5,15 @@
       <geom material="plug_black" pos="0.06 0 0" size="0.04" type="sphere"/>
       <geom material="plug_black" euler="0 1.57 0" pos="0.025 0.0 -0.0" size="0.04 0.035" type="cylinder"/>
 
-      <geom class="plug_col" pos="-0.035 0 0" size="0.036 0.021 0.021" type="box" mass=".001"/>
+      <!-- <geom class="plug_col" pos="-0.035 0 0" size="0.036 0.021 0.021" type="box" mass=".001"/>
       <geom class="plug_col" euler="0 1.57 0" pos="-0.013 0 0" size="0.045  0.007" type="cylinder" mass=".001"/>
       <geom class="plug_col" pos="0.06 0 0" size="0.04" type="sphere" mass=".001"/>
-      <geom class="plug_col" euler="0 1.57 0" pos="0.025 0.0 -0.0" size="0.04 0.035" type="cylinder" mass=".001"/>
+      <geom class="plug_col" euler="0 1.57 0" pos="0.025 0.0 -0.0" size="0.038 0.038 0.035" type="box" mass=".001"/>

```

Comment:
@avnishn  were these changes from your old branch? I didnt make these changes but if you know what they do then its no issue. They were used in the successful runs (and scripted policy tests) so i'm not too worried about leaving them here.

Reply:
@adibellathur Yeah they make grasping the peg a whole lot easier. I would leave them even, even though the scripted policy rates drop a bit.